### DISCUSSION TITLE
Selecting the plot server

### DISCUSSION THREAD

**Discussion by chuaxr** [pmlmodelling/nctoolkit#99] [Q&A] [answered]
By default, when .plot() is called, the server appears to be randomly assigned (e.g. localhost:52401). Is it possible to specify this number in advance (e.g. localhost:12345)?

**Comment by robertjwilson** [comment, marked as answer]
Hi @chuaxr 

This isn't possible right now. It should be easy to add the ability to do it though. Can you raise a feature new feature issue and specify your required functionality? https://github.com/pmlmodelling/nctoolkit/issues
